docs/content/1.documentation/2.headers/15.runtimeConfiguration.md

```diff
@@ -0,0 +1,37 @@
+# Runtime configuration
+
+:badge[Optional]{type="warning"} The headers configuration at runtime
+
+---
+
+If you need to change the headers configuration at runtime, it is possible to do it through `nuxt-security:headers` hook.
+
+## Enabling the option
+
+This feature is optionnal, you can enable it with
+
+```ts
+export default defineNuxtConfig({
+    modules: ['nuxt-security'],
+    security: {
+        runtimeHooks: true
+    }
+})
+```
+
+## Usage
+
+Within your nitro plugin. You can override the previous configuration of a route with `nuxt-security:headers`.
+
+```ts
+export default defineNitroPlugin((nitroApp) => {
+    nitroApp.hooks.hook('nuxt-security:ready', () => {
+        nitroApp.hooks.callHook('nuxt-security:headers', '/**' ,{
+            contentSecurityPolicy: {
+                "script-src": ["'self'", "'unsafe-inline'"],
+            },
+            xFrameOptions: false
+        })
+    })
+})
+```
```

Comment:
suggestion: I would recommend to move this documentation to `Usage` page (https://nuxt-security.vercel.app/documentation/getting-started/usage) to keep docs consistent